test/EndToEnd/GivenSelfContainedAppsRollForward.cs

```diff
@@ -118,25 +117,11 @@ public void WeCoverLatestNetCoreAppRollForward()
                     .Element(ns + "TargetFramework")
                     .Value;
 
-                SupportedNetCoreAppVersions.Select(v => $"netcoreapp{v[0]}")
+                SupportedNetCoreAppVersions.Versions.Select(v => $"netcoreapp{v[0]}")
                     .Should().Contain(targetFramework, $"the {nameof(SupportedNetCoreAppVersions)} property should include the default version " +
```

Comment:
Change "property" to "class" in this message now that you've refactored it (or do `{nameof(SupportedNetCoreAppVersions)}.{nameof(SupportedNetCoreAppVersions.Versions)}`)